Pass mutation_ids

hyanwong opened the pull request:
And properly explode a df with multiple sites when condense_mutations is true.

Fixes #206

kitchensjn commented:
Looks like this works other than the popup tool tip positioning is not responsive to the mouse positioning. It always has the same position near the middle of the figure in Safari (haven't checked other browsers) for both normal and condensed mutations. Don't yet know what's causing that.

hyanwong commented:
Hmm, I didn't realise that, sorry! Thanks for checking. I'll have a dig

hyanwong commented:
Fixed the bug (I changed the variable names from `function(d, i)` to `function(event, d)` to reflect the new d3 convention, but forgot to change `d` in `style("left", (d.pageX - rect.x) + "px")` to `style("left", (event.pageX - rect.x) + "px")`
